xrpl-wasm-stdlib/src/core/current_tx/mod.rs

```diff
@@ -335,24 +347,22 @@ impl CurrentTxFieldGetter for PublicKey {
 impl CurrentTxFieldGetter for Blob {
     #[inline]
     fn get_from_current_tx(field_code: i32) -> Result<Self> {
-        let mut buffer = core::mem::MaybeUninit::<[u8; 1024]>::uninit();
-        let result_code = unsafe { get_tx_field(field_code, buffer.as_mut_ptr().cast(), 1024) };
-        match_result_code(result_code, || Blob {
-            data: unsafe { buffer.assume_init() },
-            len: result_code as usize,
-        })
+        match get_variable_size_field::<1024, _>(field_code, |fc, buf, size| unsafe {
```

Comment:
Not all Blobs are the same size. Consider these changes in #62. Unclear if we should push that change first, or this one. Thoughts?

(I suppose it means Blobs will get handled like the unsigned ints above?)

Reply:
Let's see if we can change this to a generic in a future PR.

Reply:
Resolved in #62